chapter2/tests/t20/terraform/base/security_groups.tf

```diff
@@ -0,0 +1,50 @@
+module "wordpress_ec2_sg" {
+  source              = "terraform-aws-modules/security-group/aws"
+  version             = "5.1.0"
+  name                = local.labels.ec2_sg
+  description         = "Security group for the ec2 instances"
+  vpc_id              = data.aws_vpc.target.id
+  ingress_cidr_blocks = [data.aws_vpc.target.cidr_block]
+  ingress_with_cidr_blocks = [
+    {
+      rule        = "ssh-tcp"
+      cidr_blocks = local.ssh_allow_list
```

Comment:
it's better to always perfom the conversion here, than in locals.tf, because you keep the same original structure type, i.e. list and it's clean to read 

```
cidr_blocks = join(",", var.allowed_ssh_ip)
```

Reply:
> it's better to always perfom the conversion here, than in locals.tf, because you keep the same original structure type, i.e. list and it's clean to read

@dmitry-mightydevops 
Done